front-end/src/App.jsx

```diff
@@ -12,12 +12,12 @@ function App() {
   return (
     <BrowserRouter>
       <Routes>
```

Comment:
O menu vai aqui pra aparecer em todas, enquanto vc n tem página de login

```suggestion
      <Menu /> 
      <Routes>
```